mayhem/fuzz_uncurl.py

```diff
@@ -0,0 +1,42 @@
+#! /usr/bin/python3
```

Comment:
Maybe we can rename test_input to fuzzUncurl? or fuzzUncurlParse? It's good to indicate what exactly a function is doing/fuzzing. You may also be able to do something less costly/more effective when looking for chars to replace and omit from the fuzzed data. For example, you implement remove_chars but never use it? Also, it's not a great idea to hardcode the fuzzed string length, as there may be errors outside of that length that cause bugs. This needs a bit of work.

Reply:
for the fuzzed string length, would you recommend using something like sys.maxsize to indicate the size for it? If not, can you recommend another way to dynamically choose the size of the string?

Reply:
You get that for free! Fuzzing generates data of arbitrary lengths. You can certainly check to make sure that the string is less than some `max_length` if the target requires it, but I'm not sure this is the case for this target.

Reply:
ok great, I'll remove the length requirement. Are there any other things you'd like to see changed?

Reply:
That's it for now.